### ISSUE TITLE
[Windows 7] Drag & Drop doesn't work

### ISSUE THREAD

**Issue by dgroup** [evrenvarol/outlook-taskboard#9]
Hello,

I know that current version was tested on Windows 10. 
Is it possible to backport this feature to Windows 7? 

Thanks a lot for answer,
Yurii

**Comment by kohlhaas1808** [comment]
I use it on win7 with Outlook 2010 and it works like a charm.

**Comment by evrenvarol** [comment]
One of my colleagues use it on Win7 with Outlook 2013. I just didn't test it myself, but it should be working fine.  

Closing the issue, please re-open if you still have an issue.

**Comment by FredericBourgeon** [comment]
I have the issue as well. Maybe it's a setup issue.

Other thing that might be related, the "+" buttons on the taskboard do not open a template except for the one in the Backlog section.

Edit 05/11/2016:
The tasks folders I created were grouped under a group differet from the original Task folder, hence probably getting lost somewehere the script could not retrieve. I moved everything back in and carried out configuration again, it seems it's working well now. Thanks a lot for the add-in!
